test/Geometry/GeometryTestAlg.cxx

```diff
@@ -2338,6 +2344,31 @@ namespace geo{
   }
 
   //......................................................................
+  bool GeometryTestAlg::isWireAlignedToPlaneDirections
+    (geo::PlaneGeo const& plane, geo::Vector_t const& wireDir) const
+  {
+    /*
+     * Returns `true` if `wireDir` is aligned with plane frame or wire direction
+     */
+    
+    auto const isOrthogonalTo = [&wireDir](geo::Vector_t const& other)
+      { return std::abs(geo::vect::dot(wireDir, other)) < 1.0e-5; };
+    
+    // we dislike wires aligned to plane frame:
+    if (isOrthogonalTo(plane.WidthDir<geo::Vector_t>())) // || to `DepthDir()`?
+      return true;
+    if (isOrthogonalTo(plane.DepthDir<geo::Vector_t>())) // || to `WidthDir()`?
+      return true;
+    
+    // || to `GetWireDirection()`?
+    if (isOrthogonalTo(plane.GetIncreasingWireDirection<geo::Vector_t>()))
```

Comment:
I suggest removing the comments here--they are confusing.

Reply:
Son, something like:
```suggestion
    if (isOrthogonalTo(plane.WidthDir<geo::Vector_t>()))
      return true;
    if (isOrthogonalTo(plane.DepthDir<geo::Vector_t>()))
      return true;
    
    if (isOrthogonalTo(plane.GetIncreasingWireDirection<geo::Vector_t>()))
```
Let's see if I can convince GitHub to do that for me.